_posts/2017-06-27-greenpithumb.md

```diff
@@ -107,18 +76,7 @@ Lastly, we have a [12V water pump](http://amzn.to/2p90wk8), but the Raspberry Pi
 
 # Software architecture
 
-{% capture fig_img %}
-![GreenPiThumb software architecture](https://docs.google.com/drawings/d/1vY9YU9fFoyrKUh8pRe6gN0bLD1JFDq5ngkTh7yOQrOA/export/png)
-{% endcapture %}
-
-{% capture fig_caption %}
-GreenPiThumb software architecture
-{% endcapture %}
-
-<figure>
-  {{ fig_img | markdownify | remove: "<p>" | remove: "</p>" }}
-  <figcaption>{{ fig_caption | markdownify | remove: "<p>" | remove: "</p>" }}</figcaption>
-</figure>
+{% include image.html file="greenpithumb_diagram.png" alt="GreenPiThumb software architecture" fig_caption="GreenPiThumb software architecture" %}
```

Comment:
Ditto on this one. The advantage of using the docs.google.com URL is that the image updates instantly if I modify the Google Drawing.

Reply:
Absolutely, this has image has been reverted to the original external URL in 4903898